### DISCUSSION TITLE
Light Source Design Explanation

### DISCUSSION THREAD

**Discussion by srozum** [srozum/film_camera_tester#32] [Show and tell]
If you have problem understanding how to assemble the Light Unit and how it works, think about it as a flashlight. Flashlight has condenser lens in front of LED which you can move back and forth to make beam wide or narrow. As you can see, at different position of condenser lens, the surface illuminated more or less evenly.

In Light Unit, the distances between LED and Fresnel lens, and between Fresnel lens and Diffuser, selected in such way that even light beam of specific angle is generated and even portion of this beam illuminates a diffuser.

Which way fresnel lens should be oriented? Same way as condenser lens - with groves (concave part) facing forward.

Should be reflector painted white or silver? No, it will create unnecessary reflections and will affect evenness. Best case - matte black.

Will LED melt 3D printed parts or acrylic fresnel lens? During normal operation no. LED runs at 70% plus it cooled. If you going to  do experiments with it and run it at max current with no cooling, then yes.

Can you use any other diffuser? No. Some diffusers "eat" up to 4EV of light source brightness, which require LED to run brighter, higher current, hotter, etc., and light source have to be calibrated  differently.

![IMG_2673](https://user-images.githubusercontent.com/56254/232342358-4d959635-8033-476b-8758-0a2555c23c08.jpeg)
![IMG_2674](https://user-images.githubusercontent.com/56254/232342359-d15f84b7-d612-4000-86ac-c080cd3eb79a.jpeg)
![IMG_2676](https://user-images.githubusercontent.com/56254/232342360-7c323add-95dc-4fe0-98b9-a840728f7096.jpeg)

**Comment by ic-racer** [comment]
My confusion came from using fresnel on a view camera, so I had initially thought it was to be square 50radius, sandwiched right behind the diffuser. As would be the case in a view camera. Otherwise why flat?

Thanks for clearing this up. I have updated my build accordingly.

**Reply by srozum** [reply]
In enlargers and projectors condenser lens used to create even illumination. In modern LCD projectors fresnel lens used as a condenser lens for same purpose.

Acrylic fresnels smaller, lighter and cheaper than condenser lens. That's it why it was chosen for the build.

**Comment by ic-racer** [comment]
You must be some kind of engineer. Took my grandson to Carnegie Mellon two weeks ago, he is applying to engineering. Incredible the stuff he will be learning; way beyond me. Wish he were helping me with this, I think he'd enjoy it.
